services/learn-card-network/brain-service/src/routes/boosts.ts

```diff
@@ -1602,6 +1606,132 @@ export const boostsRouter = t.router({
 
             return removeBoostAsParent(parentBoost, childBoost);
         }),
+
+    sendBoostViaSigningAuthority: profileRoute
+        .meta({
+            openapi: {
+                protect: true,
+                method: 'POST',
+                path: '/boost/send/via-signing-authority/{profileId}',
```

Comment:
```suggestion
                path: '/boost/send/via-signing-authority/{profileId}',
```

I might be misunderstanding this part of the syntax, but do we need `/{profileId}` here in the URL if it's part of the body payload?

Reply:
We don't, but if I don't include it, this route conflicts with `/boosts/send/{profileId}`. I could come up with another route, but I thought this would be simplest

Reply:
Ah cool, then yes this is fine 👍